lib/ansible/modules/system/aix_lvg.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Kairo Araujo <kairo@kairo.eti.br>
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+author: "Kairo Araujo (@kairoaraujo)"
+module: aix_lvg
+short_description: Configure LVM volume groups for AIX
+description:
+  - This module creates, removes or resize volume groups on AIX LVM.
+version_added: 2.5
+options:
+  vg:
+    description:
+    - Volume Group name
```

Comment:
Please indent the array items by two characters like you did for example above in the case of the `description`. The same in all cases bellow.

Also make the `Group` lowercase to unify it across all documentation. And add period at the end of the sentence.